src/api/ready.js

```diff
@@ -0,0 +1,54 @@
+import data from '../util/data';
+import registry from '../global/registry';
+
+function ready (element) {
+  const components = registry.find(element);
+  for (let a = 0; a < components.length; a++) {
```

Comment:
New browsers get better at this, but `let a = 0, l = components.length; a < l; a++) {`?